### ISSUE TITLE
DID  created by this library can verify by third party application

### ISSUE THREAD

**Issue by Muthurajj** [decentralized-identity/did-common-java#17]

**Comment by peacekeeper** [comment]
Can you clarify what you mean with this issue?

This library does not create DIDs. It implements the DID syntax and DID document data model. If you want to create DIDs, maybe check out tools such as https://uniregistrar.io/ or https://godiddy.com/

**Comment by Muthurajj** [comment]
thank

**Comment by Muthurajj** [comment]
do you have any idea about  define type of the verification method and public in this??
https://uniregistrar.io/

**Comment by Muthurajj** [comment]
Is every DID should anchor in blockchain ??
is there any other way??

**Comment by Muthurajj** [comment]
The format used in this library can be used in did:web methods did document??

**Comment by Muthurajj** [comment]
By creating DIDs from this uniregistrar.io  or godiddy.com the DID document can be accessible by the third party verifier like dock ??

**Comment by peacekeeper** [comment]
@Muthurajj Yes, in general DIDs created by uniregistrar.io or godiddy.com should work with third-party applications. But one question is always, which DID methods does the third-party application support?

For example, if you create a `did:sov:.....` DID, but Dock doesn't support the `did:sov` method, then it will not work.

Do you know which DID methods are supported by Dock?

**Comment by Muthurajj** [comment]
Hi @peacekeeper ,
Dock use there own method did:dock

I have a doubt regarding Verifiable credentials java library can I ask it here?
